packages/rancher-gmsa-account-provider/charts/README.md

```diff
@@ -143,7 +143,36 @@ If cert-manager is not enabled, these Secrets are expected to be externally mana
 
 ### How do I identify whether the Account Provider is working as expected?
 
-TBD
+Upon installation of the Rancher gMSA Account Provider each Windows Node should have a hostProcess container running which will expose an HTTP/s API on `localhost`. This API server will listen on a port assigned to it by the host. To retrieve the port that the server is listening on, find and view the contents of the `port.txt` file located within the `/var/lib/rancher/gmsa/<ACCOUNT_PROVIDER_NAMESPACE>` directory. The API server will respond to GET requests made against the `/provider` endpoint.
+
+Before you can test the API endpoint, you must ensure that you have an impersonation account secret created within the account provider Kubernetes namespace. Once the secret has been created, you can use the following PowerShell command to test the API endpoint 
```

Comment:
> Instead of specifying you need an impersonation account Secret, we should say what is exactly required: A secret with the fields username, password, and domainName. 

I was trying to not repeat the contents of the `Create an Impersonation Account Secret` section, since it says exactly that already. I can add a link to that section so they can read in more detail there, as well as the example manifest